README.md

```diff
@@ -96,10 +94,26 @@ Copyright (c) 2012-2016 Nick Galbreath
 Licensed under the standard [BSD 3-Clause](http://opensource.org/licenses/BSD-3-Clause) open source
 license.  See [COPYING](/COPYING) for details.
 
+## BUILD TARGETS
+
+Some of the previous help runners have been merged into the Makefile. E.g.:
+
+* run-clang-asan.sh -> `make clan-asan`
+* make-ci.sh -> `make ci`
```

Comment:
Do we still need these two scripts?

Reply:
Probably not, but let's provide them until we overhaul docs.